seeg/sample_seeg_electrodes.tsv

```diff
@@ -1,102 +1,89 @@
-O'11	-33.3289	51.584	42.6881	n/a	Left-Cerebral-White-Matter	Left-Cerebral-White-Matter
-O'12	-36.4012	51.6571	43.4866	n/a	Left-Cerebral-White-Matter	Left-Cerebral-White-Matter
-O'13	-40.2868	51.7711	44.3776	n/a	ctx_lh_S_front_inf	ctx-lh-rostralmiddlefrontal
-O'14	-43.3994	52.841	44.9703	n/a	Unknown	Unknown
-O'15	-47.3633	52.958	45.8547	n/a	ctx_lh_G_front_inf-Triangul	ctx-lh-parsopercularis
-O'16	-50.3023	53.8769	47.01	n/a	ctx_lh_G_front_inf-Triangul	ctx-lh-parsopercularis
-G'1	3.83827	49.9528	64.2855	n/a	ctx_rh_G_front_sup	ctx-rh-superiorfrontal
-G'2	1.88437	50.0668	64.4318	n/a	ctx_rh_G_front_sup	ctx-rh-superiorfrontal
-G'3	-2.24201	51.2768	64.7914	n/a	Unknown	Unknown
-G'4	-5.91232	52.4179	65.0918	n/a	ctx_lh_G_front_sup	ctx-lh-superiorfrontal
-G'5	-9.09889	52.6379	65.2191	n/a	ctx_lh_G_front_sup	ctx-lh-superiorfrontal
-G'6	-12.881	52.7355	65.9931	n/a	Left-Cerebral-White-Matter	Left-Cerebral-White-Matter
-G'7	-16.0151	53.8852	66.2362	n/a	ctx_lh_S_front_sup	ctx-lh-superiorfrontal
-G'8	-19.7731	54.1198	66.3441	n/a	ctx_lh_S_front_sup	ctx-lh-rostralmiddlefrontal
-G'9	-23.0392	54.3719	66.4356	n/a	ctx_lh_S_front_sup	ctx-lh-rostralmiddlefrontal
-G'10	-26.729	55.5271	66.6815	n/a	ctx_lh_S_front_sup	ctx-lh-rostralmiddlefrontal
-G'11	-29.8758	55.7555	66.7532	n/a	Unknown	Unknown
-G'12	-33.63	56.9241	67.0007	n/a	ctx_lh_G_front_middle	ctx-lh-rostralmiddlefrontal
-G'13	-36.8267	57.1478	67.1175	n/a	Unknown	Unknown
-G'14	-39.9404	58.3136	67.3813	n/a	Unknown	Unknown
-G'15	-43.9588	58.4164	68.2296	n/a	Unknown	Unknown
-G'16	-46.777	58.5988	68.3308	n/a	Unknown	Unknown
-X'1	1.88668	21.3302	74.8493	n/a	ctx_rh_G_front_sup	ctx-rh-superiorfrontal
-X'2	-0.929053	21.1479	76.6005	n/a	Unknown	Unknown
-X'3	-4.89782	22.0368	77.9824	n/a	ctx_lh_G_front_sup	ctx-lh-superiorfrontal
-X'4	-7.92036	22.0639	79.0056	n/a	Left-Cerebral-White-Matter	Left-Cerebral-White-Matter
-X'5	-10.9436	22.0907	80.0548	n/a	Left-Cerebral-White-Matter	Left-Cerebral-White-Matter
-X'6	-13.9906	22.0987	81.1262	n/a	Left-Cerebral-White-Matter	Left-Cerebral-White-Matter
-X'7	-17.9272	22.0818	82.8182	n/a	ctx_lh_S_front_sup	ctx-lh-superiorfrontal
-X'8	-20.98	22.1185	83.8974	n/a	ctx_lh_S_front_sup	ctx-lh-superiorfrontal
-X'9	-24.8311	22.9506	85.2228	n/a	Left-Cerebral-White-Matter	Left-Cerebral-White-Matter
-X'10	-27.869	23.0116	86.2937	n/a	Left-Cerebral-White-Matter	Left-Cerebral-White-Matter
-X'11	-30.8945	23.0629	87.3368	n/a	ctx_lh_S_precentral-sup-part	ctx-lh-precentral
-X'12	-33.8561	23.8809	88.514	n/a	Unknown	Unknown
-X'13	-37.9086	24.008	89.7199	n/a	Unknown	Unknown
-X'14	-40.8264	24.8544	90.8372	n/a	Unknown	Unknown
-X'15	-43.814	24.9309	91.8615	n/a	Unknown	Unknown
-X'16	-48.7736	24.9923	93.7919	n/a	Unknown	Unknown
-EKG1	n/a	n/a	n/a	n/a	n/a	n/a
-EKG2	n/a	n/a	n/a	n/a	n/a	n/a
-EVENTS/MARKERS	n/a	n/a	n/a	n/a	n/a	n/a
\ No newline at end of file
+name	x	y	z	destriuex	desikan-killiany
+L'1	-1.095576	66.539884	35.287211	Unknown	Unknown
```

Comment:
And then instead of these values in the subject's MRI space, store the values in MNI space.

Then the example can be "hey I have coordinates only in MNI space, how do I use them?" and the answer is "with fsaverage because fsaverage is in MNI space".